datascroller/scroller.py

```diff
@@ -357,8 +366,14 @@ def scroll(scrollable):
         print('type ' + str(type(scrollable)) + ' not yet scrollable!')
 
 
-def scroll_csv(csv_path):
-    pandas_df = pd.read_csv(csv_path, dtype=object)
+def scroll_csv(csv_path,
+               sep,
+               encoding):
+
+    pandas_df = pd.read_csv(csv_path,
```

Comment:
Would it make sense to wrap this in a try/catch block? Just so we can handle `ValueError` and `UnicodeError` more gracefully if we want?

Reply:
I might at the parsing step. By the time you get here, we assume the call is clean because, well, the last step is supposed to validate it. If there is a further error internal to pandas I'm not sure I'd change the error message.

Reply:
Makes sense. I guess I'm thinking more of a valid call of the function but where the arguments don't match the file. I guess seeing the full stack trace from pandas isn't the worst thing, but we might prevent a few headaches if we write our own. But you're right, maybe that's counterproductive.